Sources/Commands/SwiftPackageTool.swift

```diff
@@ -183,7 +183,24 @@ public struct SwiftPackageTool: SwiftTool {
                     return try get(manifest, manifestParser: parseManifest)
                 }
             }
-        
+
+            func findDirectoryReferences(path: String) throws -> [String] {
+                let rootDirectories = walk(path, recursively: false)
+                let rootDirectoriesToConsider = rootDirectories.filter {
+                    if $0.hasSuffix(".xcodeproj") { return false }
+                    if $0.hasSuffix(".playground") { return false }
+                    if $0.basename.hasPrefix(".") { return false }
+                    return $0.isDirectory
```

Comment:
I would like to hear your thoughts about this exclude list. I based it on [`Package.shouldConsiderDirectory`](https://github.com/apple/swift-package-manager/blob/83f0238bb9e30d21f49baf1c5ef5770dd0cb9c8b/Sources/PackageLoading/PackageExtensions.swift#L268-L279), but I'm not sure if it's complete enough for this use case.